examples/nfdump-wrapper.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+TMPFILE=$(mktemp)
+cat - > "$TMPFILE"
+nfpcapd -r "$TMPFILE" -l .
+rm "$TMPFILE"
+for file in nfcapd.*
+do
+  nfdump -r $file -o csv | head -n -3 | zq -i csv -f ndjson - > ${file}.ndjson
```

Comment:
Any reason to prefer writing these to files rather than straight to standard output?
```suggestion
  nfdump -r $file -o csv | head -n -3 | zq -i csv -f ndjson -
```

Reply:
@nwt: My understanding is that Brimcap expects to get its input by tailing files found in the analyzer's working directory. So whether Brimcap ends up trying to load all the log files found there or a subset specified via `globs:`, I've been led to believe that they have to exist and be discovered as named files. Just to make sure I hadn't missed a memo, I tested it with your variation where the NDJSON just gets sent to stdout and indeed there's an error:

```
$ /opt/Brim/resources/app.asar.unpacked/zdeps/brimcap load -root "$HOME/.config/Brim/data/brimcap-root" -config nfdump-noah.yml -p testpool4 ~/wrccdc.pcap 
100.0% 500.0MB/500.0MB records=0 
status code 500: pool "testpool4": internal error: serialize transaction: empty transaction
```

Perhaps @mattnibs might chime in if there's more to be said here.

Reply:
Ah, right. I have no idea what I was thinking when I made that suggestion.